lib/modules/manager/devcontainer/extract.ts

```diff
@@ -43,6 +47,39 @@ export function extractPackageFile(
           featureDep.depType = 'feature';
           featureDep.pinDigests = false;
           deps.push(featureDep);
+          if (!value.version) {
+            continue;
```

Comment:
We should return here a dependency with skipReason `no-version`.
So that they are showing up in the logs, reports and such